packages/ui/src/logo/agreeto.tsx

```diff
@@ -0,0 +1,221 @@
+        fill="#EEC6B3"
+      />
+      <path
+        d="M682.593 207.343C656.683 207.343 620.65 189.124 601.217 179.408L597.573 177.788C576.52 167.262 526.723 74.144 526.318 73.3346C524.699 70.5001 524.293 68.4758 525.104 67.2615C525.508 66.4517 526.318 66.0469 527.533 66.0469C536.844 66.0469 568.423 85.4799 583.808 95.6015C596.764 104.104 629.962 121.918 656.683 135.683C670.447 142.97 682.189 149.043 689.071 153.091C710.931 165.237 728.343 184.266 725.1 192.363C721.864 200.46 719.033 200.865 693.929 205.723L691.095 206.128C688.262 206.937 685.427 207.343 682.593 207.343Z"
+        fill="#D3A690"
+      />
+      <path
+        d="M526.316 72.9386C526.316 72.9386 576.114 166.866 597.572 177.392C618.625 187.918 663.564 211.4 691.094 206.136C718.221 200.874 721.458 200.874 724.694 192.372C727.937 183.87 709.714 164.841 689.07 153.1C668.422 141.359 603.239 108.566 583.806 95.6102C564.373 82.6548 514.981 52.6953 526.316 72.9386Z"
+        fill="#EEC6B3"
+      />
+      <path
+        d="M309.392 223.199L310.607 222.389C314.655 219.96 318.299 217.936 319.513 217.531C321.133 217.531 323.157 218.746 326.396 221.175C319.918 221.985 314.25 222.389 309.392 223.199Z"
+        fill="#EEC6B3"
+      />
+      <path
+        d="M309.393 223.613C314.657 222.398 320.325 221.993 325.993 221.589C322.754 219.16 320.324 217.54 319.11 217.945C317.895 218.35 314.252 220.779 309.393 223.613Z"
+        fill="#EEC6B3"
+      />
+      <path
+        d="M482.029 218.031H341.336V301.168H482.029V218.031Z"
+        fill="url(#pattern2)"
+      />
+      <path
+        d="M428.745 277.383C414.98 277.383 367.612 242.97 344.13 220.703H434.008C444.13 232.039 452.226 241.756 455.061 246.614C458.704 252.282 459.11 257.545 456.68 262.403C452.632 270.905 439.676 276.573 429.555 277.383C429.15 277.383 428.745 277.383 428.745 277.383Z"
+        fill="#D3A690"
+      />
+      <path
+        d="M429.15 277.383C442.915 276.169 465.992 265.238 454.656 246.614C451.822 242.161 444.13 232.444 434.008 220.703H344.13C368.421 243.78 416.599 278.598 429.15 277.383Z"
+        fill="#EEC6B3"
+      />
+      <path
+        d="M279.081 234.257C278.675 233.447 278.27 232.638 278.27 231.423C278.675 216.848 317.137 193.366 338.999 216.038C340.619 217.658 342.238 219.277 344.262 221.302H434.141C401.347 184.459 340.214 124.54 302.967 117.658C253.575 108.751 99.7625 363.207 66.1592 423.936C32.556 484.664 46 814.354 46 814.354C169.482 813.948 116.732 814.417 116.732 814.417L166.935 661.382C166.935 661.382 187.583 658.548 218.352 651.666V334.662C218.352 290.937 243.048 253.286 279.081 234.257Z"
+        fill="#EEC6B3"
+      />
+      <defs>
+        <filter
+          id="filter0_d_2007_70811"
+          x="236.327"
+          y="209.474"
+          width="724.791"
+          height="724.791"
+          filterUnits="userSpaceOnUse"
+          color-interpolation-filters="sRGB"
```

Comment:
this doesn't work in react I think. It expects camelCase.

also in a couple of other places.
![CleanShot 2022-11-24 at 20 05 26](https://user-images.githubusercontent.com/18185649/203851768-ad82c94f-1ec7-4fb0-86f3-1f84a8d3e967.png)

![CleanShot 2022-11-24 at 20 05 19](https://user-images.githubusercontent.com/18185649/203851791-2674afbf-c5b4-4e90-9738-b7ec9c1f9da7.png)

Reply:
weird that svg worked for me 🤔